### DISCUSSION TITLE
[Question]: Get non-specific object motion snapshot

### DISCUSSION THREAD

**Discussion by leonbrag** [blakeblackshear/frigate#13383] [General Support]
### Describe the problem you are having

From reading all the discussions and documentations, my understanding that Frigate is really all about detecting objects.

It detects area of change, then passes to object detection pipeline. If area's change cannot quantify as known object, Frigate treats it as motion. 

I want to write an automation that sends me snapshot **when motion is detected** (in addition to when **specific object identified**).  

Scenario:
1. Wild animal that cannot be quantified by model walks by, rock is thrown in the window, etc 

What is a right way to do it?

I tried getting On notification using: frigate/<camera_name>/motion/set
Getting time MQTT timestamp, converting to epoch
Use timline API to get snapshot(s) withing this timeline. 

However approach above did not work, nothing was return. Even though I can see motion in the UI

What is a right way to get notified with the scenario above?
Note such notification should only be for something that does not qualify as alert/detection. These notifications will come via reviews topic. 

Thanks

### Version

.14

### Frigate config file

```yaml
na
```

### Relevant Frigate log output

```shell
na
```

### Relevant go2rtc log output

```shell
na
```

### FFprobe output from your camera

```shell
na
```

### Operating system

Debian

### Install method

Docker Compose

### Object Detector

OpenVino

### Network connection

Wired

### Camera make and model

na

### Any other information that may be helpful

_No response_I

**Comment by NickM-27** [comment]
listen on the `: frigate/<camera_name>/motion` topic and send a notification

**Reply by leonbrag** [reply]
Sorry not following 

The frigate/<camera_name>/motion MQTT message has only ON/OFF in the payload.  That what I saw in the message.

Say I get one ON message, do I then turn around in inject my own message to frigate/notification topic ?  If so, where would I get all the data to construct proper payload ?

The objective is to get notification id of the event (or something) that allows me to pull snapshot at the moment ON message was triggered.

What I am trying to mimic is something like frigate/motion topic. that get events that are NOT detections or alerts.

**Reply by NickM-27** [reply]
What you are trying to do isn't possible. There will be no event ID, events are for objects only. You can send the latest frame from the camera when the topic goes to ON but that is all there is to it.

**Reply by leonbrag** [reply]
Oh, you mean, in the on notification call GET /api/<camera_name>/latest.jpg to retrieve latest image?

If so, this is perfect!!! 

Brilliant. Exactly what I need. Thanks!!

And just to double check, frigate/<camera_name>/motion will NOT be fired for Alert/Detection that is fired on review topic, right ?
